Keas.Mvc/ClientApp/components/Equipment/EquipmentBigFixInfo.tsx

```diff
@@ -0,0 +1,121 @@
+import * as PropTypes from "prop-types";
+import * as React from "react";
+import { Button, Modal, ModalBody, ModalFooter, ModalHeader, Table } from "reactstrap";
+
+interface IProps {
+    bigfixId: string;
+}
+
+interface IState {
+    bigfixModal: boolean;
+    computerInfo: object;
+    isFetched: boolean;
+}
+
+export default class EquipmentDetails extends React.Component<IProps, IState> {
```

Comment:
this should be `EquipmentBigFixInfo`, not `EquipmentDetails`

Reply:
fixed